Add Satont/twitch-notifier (#16)
grammyjs/awesome-grammY: README.md

- [grammyjs/docs-bot](https://github.com/grammyjs/docs-bot) - Search and share grammY docs right inside Telegram. [ᴜsᴇ](https://t.me/grammydocsbot)
- [@ChannelHashBot](https://t.me/ChannelHashBot) - Keep track of hashtags that are sent to your group by forwarding them to a channel.
- [@cattpbot](https://github.com/KnorpelSenf/cattpbot) - Send cat pictures from https://http.cat instead of HTTP status codes. [ᴜsᴇ](https://t.me/cattpbot)

> - [Satont/twitch-notifier](https://github.com/Satont/twitch-notifier) - Get notifier about Twitch translations starts, ends, update. [ᴜsᴇ](https://t.me/TwiNotifyBot)

Reviewer: ```suggestion
- [Satont/twitch-notifier](https://github.com/Satont/twitch-notifier) - Get notified about Twitch translation start, end and updates. [ᴜsᴇ](https://t.me/TwiNotifyBot)
```

Author: Thank you!

> - [Satont/twitch-notifier](https://github.com/Satont/twitch-notifier) - Get notified about Twitch translations starts, ends, update. [ᴜsᴇ](https://t.me/TwiNotifyBot)

Reviewer: ```suggestion
- [Satont/twitch-notifier](https://github.com/Satont/twitch-notifier) - Get notified when a Twitch broadcast starts, ends or updates. [ᴜsᴇ](https://t.me/TwiNotifyBot)
```
